src/android/AudioManagement.java

```diff
@@ -116,10 +142,7 @@ private void notifyActionError(CallbackContext callbackContext, String errorMess
 
     private void getNotificationPolicyAccessState(CallbackContext callbackContext) throws JSONException {
         JSONObject result = new JSONObject();
-        boolean granted = true;
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
@ephelsa Any reason we removed this Android version based check? I understand Android 6 and below is not in use anymore but we could leave this code as is.

Reply:
Looks fine